sensor/common/metrics/metrics.go

```diff
@@ -285,6 +285,22 @@ var (
 		},
 		[]string{"central_id", "hosting", "install_method", "sensor_id"},
 	)
+
+	// responsesChannelSize a gauge to track the responses channel size
+	responsesChannelSize = prometheus.NewGauge(prometheus.GaugeOpts{
+		Namespace: metrics.PrometheusNamespace,
+		Subsystem: metrics.SensorSubsystem.String(),
+		Name:      "responses_channel_size",
```

Comment:
I would expect this metric with its current name to always show the value set in `ROX_RESPONSES_CHANNEL_BUFFER_SIZE`.
I would suggest to rename it to something more descriptive `num_messages_waiting_for_transmission_to_central`.